playbook-local.yml

```diff
@@ -14,6 +14,7 @@ ui:
   supplemental_files: ./dsc-style-bundle/supplemental-files/rancher
 asciidoc:
   attributes:
+    latest-version: 'v1.0.0'
```

Comment:
This needs to be added to the playbook-remote.yml too, otherwise it'll cause issues if someone builds the remote version. In general, these two playbook files should be in sync outside of the content source config portion.